Dockerfile

```diff
@@ -1,18 +1,37 @@
-FROM quay.io/3scale/ruby:2.1
-MAINTAINER Oriol Martí <oriol@3scale.net> # 2015-06-26
+FROM centos:7 
+MAINTAINER Daniel Cesario <dcesario@redhat.com>
 
-RUN apt-install -y libssl-dev
+USER root
+
+RUN yum -y update \
+	&& yum install -y centos-release-scl \
+	&& yum install -y rh-ruby23 rh-ruby23-ruby-devel \
+	&& yum -y install make gcc \
+	&& yum -y install openssl-devel 
 
 WORKDIR /opt/echo-api/
 
 COPY ./ /opt/echo-api
+COPY contrib/scl_enable /opt/echo-api/etc/
+
+RUN chown -fR 1001:1001 /opt/echo-api
+
+RUN	source /opt/echo-api/etc/scl_enable \
+	&& gem install -N bundler \
```

Comment:
always good to install a specific version with `--version=X`
the latest is `1.14.3` IIRC